cmp/grid/columns/Column.js

```diff
@@ -317,7 +317,9 @@ export class Column {
                 colId: this.colId,
                 headerValueGetter: (agParams) => {
                     return agParams.location === 'header' ?
-                        isFunction(headerName) ? headerName({column: this, gridModel, agParams}) : headerName :
+                        isFunction(headerName) ?
```

Comment:
why not checking to see if this is not a string -- couldn't be an element, or a function as well?

Also, this should default to 'displayName', not chooserName -- that seems tailor made for that!